### PR TITLE
add makefile, dockerfile so it's easily usable

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+IMAGE_NAME := koji-build-finder
+
+build:
+	docker build --tag=$(IMAGE_NAME) .
+
+shell:
+	docker run -v $(PWD):/src -w /src -it $(IMAGE_NAME) bash
```
Comment (rnc) [inline Makefile]: I had to use `docker run -v $(CURDIR):/src` as PWD doesn't not work for me. I think you could also use `$(shell pwd)`

File: README.md [hunk 2]
```diff
@@ -20,6 +20,20 @@ The format of the reports is currently either text or HTML.
 
 ## Usage
 
+There is a Dockerfile and a Makefile supplied in this repo. All you need to do is:
+
```
Comment (rnc) [inline README.md]: I think I would prefer this in a separate section to the usage as its not required to use Docker to build or run the tool.

File: README.md [hunk 3]
```diff
@@ -20,6 +20,20 @@ The format of the reports is currently either text or HTML.
...
+1. Build the container image:
+  ```
+  $ make build
+  ```
+
+2. Get shell in the container abd play with the tool:
+  ```
+  $ make shell
+  [root@bd9c05062b1a src]# java -jar target/koji-build-finder-1.0.0-SNAPSHOT-jar-with-dependencies.jar
+  ```
```
Comment (rnc) [inline README.md]: I had to run `chcon -Rt svirt_sandbox_file_t <pwd>` in order to get the mount working correctly.

**Comment by rnc** [comment]
Thanks for the pull request!

File: Makefile (hunk 1 above)
Comment (TomasTomecek) [reply Makefile]: can do, probably a difference in make version

File: README.md (hunk 3 above)
Comment (TomasTomecek) [reply README.md]: I always forget the :Z flag, silly me.

File: README.md (hunk 2 above)
Comment (TomasTomecek) [reply README.md]: my point with this PR is to make it easy for people to use the tool right away; especially for folks who have zero java experience, such my myself -- I literally had to google how to compile java sources; I can write about that and put it in a different section, that's not a problem

**Comment by TomasTomecek** [comment]
I hope I addressed all your comments.

**Pull request merged**